src/ops/execute/execute_test.go

```diff
@@ -66,19 +67,19 @@ var _ = Describe("installerArgs", func() {
 	)
 
 	It("Returns the correct list with no extra args", func() {
-		args := installerArgs(ignitionPath, device, nil)
+		args := utils.InstallerArgs(ignitionPath, device, nil)
 		expected := []string{"install", "--insecure", "-i", "/tmp/ignition.ign", "/dev/sda"}
 		Expect(args).To(Equal(expected))
 	})
 
 	It("Returns the correct list with empty extra args", func() {
-		args := installerArgs(ignitionPath, device, []string{})
+		args := utils.InstallerArgs(ignitionPath, device, []string{})
 		expected := []string{"install", "--insecure", "-i", "/tmp/ignition.ign", "/dev/sda"}
 		Expect(args).To(Equal(expected))
 	})
 
 	It("Returns the correct list with extra args", func() {
-		args := installerArgs(ignitionPath, device, []string{"-n", "--append-karg", "nameserver=8.8.8.8"})
+		args := utils.InstallerArgs(ignitionPath, device, []string{"-n", "--append-karg", "nameserver=8.8.8.8"})
 		expected := []string{"install", "--insecure", "-i", "/tmp/ignition.ign", "-n", "--append-karg", "nameserver=8.8.8.8", "/dev/sda"}
```

Comment:
These tests don't belong here, they have nothing to do with execute.
Consider moving to utils_test/ops_test